### DISCUSSION TITLE
Ability to set playhead for PlaybackProcessor

### DISCUSSION THREAD

**Discussion by jtag05** [DBraun/DawDreamer#174] [Q&A]
I'm curious if there is a way to set the playhead for a `PlaybackProcessor` in order to only render a specific chunk of an input file after running it through a set of processors that require the full original input.

**Comment by DBraun** [comment]
I think PlaybackWarpProcessor has better support for this. You could follow this example https://github.com/DBraun/DawDreamer/blob/1e15527fd35f5d6125e1105a40ff79aaf5180f71/tests/test_playbackwarp_processor.py#L113 
but set
`drums.warp_on = False` It should be off by default if there's no `.asd` file.
If you did
`drums.set_clip_positions([[4., 9., 0.]])`
it would mean start using the audio file's audio at 4 beats into the rendering. You'll stop hearing the audio at 9 beats (so duration of 5 beats). The third value 0 is not used yet in this situation because `warp_on` is False. I have to make a better decision about what it should mean. It should probably be an offset measured in seconds. Whereas if `warp_on` is True it's supposed to be an offset measured in beats like the first two values.
